### ISSUE TITLE
[FEATURE] Indicate that NSThirdPartyExceptionAllowsInsecureHTTPLoads allows insecure HTTP loads as it's indicated with NSExceptionAllowsInsecureHTTPLoads

### ISSUE THREAD

**Issue by carlospolop** [MobSF/Mobile-Security-Framework-MobSF#1375]
Analysing an IPA I noticed that Mobsf indicates the URLs that can be loaded with HTTP when the key `NSExceptionAllowsInsecureHTTPLoads` is present, but it doesn't indicate the URLs when the key `NSThirdPartyExceptionAllowsInsecureHTTPLoads` is present.
According to this: http://www.neglectedpotential.com/2015/06/working-with-apples-application-transport-security/  both keys mean the same.

Maybe you could also indicate as dangerous which domains can be accessed using HTTP when the key `NSThirdPartyExceptionAllowsInsecureHTTPLoads` is present.

**Comment by ajinabraham** [comment]
Can you share a sample plist file that I can use to reproduce this?

**Comment by carlospolop** [comment]
Sorry man, I can give you the full plist but hopefully this information is enough for you:
```
	<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
		<key>NSExceptionDomains</key>
		<dict>
			<key>domain.net</key>
			<dict>
				<key>NSIncludesSubdomains</key>
				<true/>
				<key>NSRequiresCertificateTransparency</key>
				<false/>
				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
				<true/>
				<key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
				<false/>
			</dict>
			<key>domain2.net</key>
			<dict>
				<key>NSIncludesSubdomains</key>
				<true/>
				<key>NSRequiresCertificateTransparency</key>
				<false/>
				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
				<true/>
				<key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
				<false/>
			</dict>
			<key>https://some.domain2.net</key>
			<dict>
				<key>NSExceptionAllowsInsecureHTTPLoads</key>
				<true/>
			</dict>
			<key>https://some2.domain2.com</key>
			<dict>
				<key>NSExceptionAllowsInsecureHTTPLoads</key>
				<true/>
			</dict>
			<key>localhost</key>
			<dict>
				<key>NSExceptionAllowsInsecureHTTPLoads</key>
				<true/>
			</dict>
			<key>domain3.net</key>
			<dict>
				<key>NSRequiresCertificateTransparency</key>
				<false/>
				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
				<true/>
				<key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
				<false/>
			</dict>
		</dict>
	</dict>
```
When the key `NSThirdPartyExceptionRequiresForwardSecrecy` is used MobSF doesn't indicate it

**Comment by ajinabraham** [comment]
fixed in 3.0.9
